addons/mass_mailing/models/mailing.py

```diff
@@ -178,6 +178,7 @@ def _get_default_mail_server_id(self):
         help='If checked, recipients will be mailed only once for the whole campaign. '
              'This lets you send different mailings to randomly selected recipients and test '
              'the effectiveness of the mailings, without causing duplicate messages.')
+    ab_testing_is_winner_mailing = fields.Boolean()
```

Comment:
shouldn't we store this information on the campaign ?
(and `ab_testing_is_winner_mailing` will be a computed based on the field on the campaign)

because from a structure POV, you can technically have multiple winner for the same campaign